sonar-xml-plugin/pom.xml

```diff
@@ -163,7 +159,7 @@
               <rules>
                 <requireFilesSize>
                   <maxsize>2600000</maxsize>
```

Comment:
The idea of this check is to keep it as close to real weight as possible, so if it's now less than 2.5 Mb it's nice to update `maxsize` too